Resources/Prototypes/Nyanotrasen/Loadouts/Jobs/Wildcards/gladiator.yml

```diff
@@ -9,3 +9,8 @@
   id: ClothingOuterArmorGladiator
   equipment:
     outerClothing: ClothingOuterArmorGladiator
+
+- type: loadout # Umbra
+  id: GladiatorShoes
+  equipment:
+    shoes: ClothingShoesSandals
```

Comment:
put it back in _Umbra

Reply:
either way dont add things to nyano its exclusively legacy

if theres changes made add comments in _Unbra